state/machine.go

```diff
@@ -2069,7 +2070,7 @@ func (m *Machine) CreateUpgradeSeriesPrepareLock() error {
 		}
 		locked, err := m.IsLocked()
 		if err != nil {
-			return nil, errors.Wrap(err, jujutxn.ErrNoOperations)
+			return nil, errors.Trace(err)
 		}
 		if locked {
 			return nil, errors.AlreadyExistsf("upgrade series prepare lock for machine %q", m)
```

Comment:
since the doc is already there... shouldn't we return jujutxn.ErrNoOperations?

Reply:
Whats the best way to indicate that there are no operations to be run but because the item already exists. I would like the client to receive an Already Exists error. What is best here?